### PR TITLE
fix bug 1690034 - Add Rust OOM stuff to the irrelevant signature list.

### PR DESCRIPTION
**Pull request by amccreight**

### CODE REVIEW

File: socorro/signature/siglists/irrelevant_signature_re.txt [hunk 1]
```diff
@@ -93,6 +94,7 @@ panic_abort::
 PR_WaitCondVar
 RaiseException
 RealMsgWaitFor
+__rust_alloc_error_handler
```
Comment (willkg) [inline socorro/signature/siglists/irrelevant_signature_re.txt:97]: This has two underscores at the beginning, but I see both `_rust_alloc_error_handler` and `__rust_alloc_error_handler` in crash reports, so I'd add both versions.

Example with single underscore: bp-e17483cd-347e-46fb-ab61-fdc320210131

Example with double underscore: 11f6b902-efa9-4861-9781-70d4d0210201

Comment (willkg) [reply socorro/signature/siglists/irrelevant_signature_re.txt:97]: Oh, you did this by editing the file in GitHub. I'll land this and then add the second part.

**Comment by willkg** [comment]
```
app@socorro:/app$ socorro-cmd signature bp-e17483cd-347e-46fb-ab61-fdc320210131 bp-68d7923e-1dbc-4346-824e-ebd0b0210201
Crash id: e17483cd-347e-46fb-ab61-fdc320210131
Original: OOM | large | mozalloc_abort | alloc::alloc::__alloc_error_handler::__rg_oom
New:      OOM | large | mozalloc_abort | _rust_alloc_error_handler
Same?:    False
Crash id: 68d7923e-1dbc-4346-824e-ebd0b0210201
Original: OOM | large | mozalloc_abort | alloc::alloc::__alloc_error_handler::__rg_oom
New:      OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::with_capacity
Same?:    False
```

I'm going to add `_rust_alloc_error_handler` (single double-score) in a follow-up PR.

**Review by willkg — APPROVED** [review]

**Comment by willkg** [comment]
Thank you!

**Comment by amccreight** [comment]
Thanks for the followup! I guess my sample size of 3 crash reports was insufficient.

**Pull request merged**